src/util/Util.js

```diff
@@ -371,22 +371,16 @@ class Util {
   }
 
   /**
-   * Data that can be resolved to give a string. This can be:
-   * * A string
-   * * An array (joined with a new line delimiter to give a string)
-   * * Any value
-   * @typedef {string|Array|*} StringResolvable
-   */
-
-  /**
-   * Resolves a StringResolvable to a string.
-   * @param {StringResolvable} data The string resolvable to resolve
+   * Verifies the provided data is a string, otherwise throws provided error.
+   * @param {string} data The string resolvable to resolve
+   * @param {Error} [error=Error] The error to throw, should the validation fail
+   * @param {boolean} [allowEmpty=true] Wether an empty string should be allowed
    * @returns {string}
    */
-  static resolveString(data) {
-    if (typeof data === 'string') return data;
-    if (Array.isArray(data)) return data.join('\n');
-    return String(data);
+  static verifyString(data, error = new Error(), allowEmpty = true) {
```

Comment:
Just a small tweak
```suggestion
  static verifyString(data, error = new Error(`Expected a string, got ${data} instead`), allowEmpty = true) {
```

Reply:
Psst.
Just a small tweak
```suggestion
  static verifyString(data, error = new Error(`Expected a string, got ${data} instead`), allowEmpty = true) {
```